iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/ThreeWindingsTransformerXml.java

```diff
@@ -41,10 +48,15 @@ protected boolean hasSubElements(ThreeWindingsTransformer twt) {
                 || twt.getLeg3().hasRatioTapChanger()
                 || twt.getLeg1().hasPhaseTapChanger()
                 || twt.getLeg2().hasPhaseTapChanger()
-                || twt.getLeg3().hasPhaseTapChanger()
-                || twt.getLeg1().getCurrentLimits() != null
-                || twt.getLeg2().getCurrentLimits() != null
-                || twt.getLeg3().getCurrentLimits() != null;
+                || twt.getLeg3().hasPhaseTapChanger();
```

Comment:
I would put the content of `hasSubElements(twt)` inside `hasSubElements(twt, context)` as it's again a bit confusing to get a possibly different result if you forget to provide the context